synfig-studio/src/gui/docks/dock_info.cpp

```diff
@@ -157,30 +157,30 @@ studio::Dock_Info::Dock_Info()
 	// Render Progress Bar
 	Gtk::HBox *render_box = manage(new Gtk::HBox());
 	Gtk::Label *render_progress_label = manage(new Gtk::Label());
-	Gtk::Label *left_spacer_label = manage(new Gtk::Label());
+	Gtk::Overlay *overlay = manage(new Gtk::Overlay());
+
 	render_progress_label->set_markup(etl::strprintf("<b>%s</b>", _("Render Progress:")));
-	grid->attach_next_to(*render_progress_label, *separator2, Gtk::POS_BOTTOM, 8, 1);
 
 	// Render progress CSS ID
 	render_progress.set_name("render-progress");
 	render_progress.set_hexpand(true);
 	render_progress.set_vexpand(false);
-	render_progress.set_show_text(true);
-	render_progress.set_text(strprintf("%.1f%%", 0.0));
+	render_progress.set_margin_end(5);
+	render_progress.set_valign(Gtk::ALIGN_CENTER);
 	render_progress.set_fraction(0.0);
 
+	render_percentage.set_text(strprintf("%.1f%%", 0.0));
+
+	overlay->add_overlay(render_progress);
+	overlay->add_overlay(render_percentage);
```

Comment:
Shouldn't `render_progress` be added to a `Gtk::Container`? According to [GTKMM 3.24 docs](https://developer.gnome.org/gtkmm/3.24/classGtk_1_1Overlay.html#a6e8eb1201093c8b9c0ea7f0f274ae587) on `add_overlay()`:

> The widget will be stacked on top of the main widget added with **Gtk::Container::add()**.

@ice0's example in this https://github.com/synfig/synfig/issues/2037#issuecomment-808842732 also adds the progress to a container and overlays the label on top of that progress bar
```python
...
    ...
        overlay.add(self.progressbar)

        self.plabel = Gtk.Label.new()
        overlay.add_overlay(self.plabel)
    ...
...
```

GTK YouTuber also does the same [Tutorial 3 at timestamp 3:35](https://youtu.be/Jj0sXiJ0ENU?t=335)